object_manager.go

```diff
@@ -304,6 +305,40 @@ func (objMgr *ObjectManager) GetFixedAddress(netview string, cidr string, ipAddr
 	return &res[0], nil
 }
 
+func (objMgr *ObjectManager) UpdateFixedAddress(netview string, cidr string, ipAddr string, macAddress string, vmID string) (*FixedAddress, error) {
```

Comment:
I don't think this code would work if user just wanted to update vmID but not macAddress.